lib/engine/config/game/g_18_co.rb

```diff
@@ -111,24 +111,24 @@ module G18CO
 		"28": 2,
 		"29": 2,
 		"co8": {
-			"count": 1,
+			"count": 5,
 			"color": "green",
-			"code": "town=revenue:20;junction;path=a:0,b:_0;path=a:1,b:_0;path=a:3,b:_0"
+			"code": "town=revenue:20,to_city:1;junction;path=a:0,b:_0;path=a:1,b:_0;path=a:3,b:_0"
```

Comment:
I'm not sure the 'to_city' should be part of 'town' since a tile can have multiple towns on it.
Maybe this should be part of 'upgrade' instead

Reply:
Being a flag on town would allow a potential for some weird tile where one town upgrades to a city yet a different town stays a town. I don't know of such a case but I don't see why multiple towns precludes the option from being attached to the town.

Reply:
I suppose... I'll leave it to Toby to decide.

Reply:
Phrased another way, its indicating "This town is eligible to convert to a city" rather than "This tile is eligible to convert to a city".

Reply:
yes, it's more flexible in that case, but a bit more verbose

Reply:
Toby, I'm unsure if you're encouraging we move the flag to upgrade. I see how to write the code, however the more I think about it, the more I prefer having it on town because it is more specific. For example, if there were no towns on the tile I'm not sure what the expected usage of the flag would be.

Reply:
i'll leave that decision to you but it seems like town is fine